### PR TITLE
News Tab Created NYTimes API added

### PR DESCRIPTION
**Pull request by vahgar**
Hey, I have added News Tab. I will further contribute more. It will be a good gesture if you accept my pull request. Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -210,6 +210,13 @@ A collective list of JSON APIs for use in web development.
 | Soundcloud | Music | No | [Go!](https://developers.soundcloud.com/) |
 | Spotify | Music | Parts | [Go!](https://developer.spotify.com/web-api/) |
 
+
+### NEWS
```
Comment (mikestreety) [inline README.md]: Could you make News title case please?

File: README.md [hunk 2]
```diff
@@ -210,6 +210,13 @@ A collective list of JSON APIs for use in web development.
...
+
+| API | Description | OAuth | Link |
+|---|---|---|---|
+| New York Times | Provides News | No, but apikey is required | [Go!](https://developer.nytimes.com/) |
```
Comment (mikestreety) [inline README.md]: Remove capital `N` on news

Comment (mikestreety) [inline README.md]: Could you put `apikey` in code blocks please?

**Comment by vahgar** [comment]
I have changed things accordingly.

**Pull request merged**